cpp_linter/rest_api/github_api.py

```diff
@@ -229,19 +229,21 @@ def make_annotations(
                 output += f" does not conform to {style_guide} style guidelines. "
                 output += "(lines {lines})".format(lines=", ".join(line_list))
                 log_commander.info(output)
-        for note, file in zip(tidy_advice, files):
-            for n in note:
-                if n.filename == file.name:
+        for concern, file in zip(tidy_advice, files):
+            for note in concern.notes:
+                if note.filename == file.name:
                     output = "::{} ".format(
-                        "notice" if n.note_type.startswith("note") else n.note_type
+                        "notice"
+                        if note.severity.startswith("note")
+                        else note.severity
                     )
                     output += "file={file},line={line},title={file}:{line}:".format(
-                        file=file.name, line=n.line
+                        file=file.name, line=note.line
                     )
                     output += "{cols} [{diag}]::{info}".format(
-                        cols=n.cols,
-                        diag=n.diagnostic,
-                        info=n.note_info,
+                        cols=note.cols,
+                        diag=note.diagnostic,
+                        info=f"{note.rationale} -- see {note.diagnostic_link}",
```

Comment:
Apparently, Markdown syntax isn't rendered in these annotations' body:
![image](https://github.com/cpp-linter/cpp-linter/assets/14963867/7d90cde3-3a27-41db-8c4c-d871a1c0a2da)

I have to revert this change

Reply:
reverted in https://github.com/cpp-linter/cpp-linter/commit/a32274920375375c717ae09abc67a47bc79c6bfc